wrappers/kv_log.go

```diff
@@ -0,0 +1,436 @@
+package wrappers
+
+import (
+	"time"
+
+	"github.com/portworx/kvdb"
+	"github.com/sirupsen/logrus"
+)
+
+type kvLogger struct {
+	// kv is the kvdb which is wrapped
+	kv kvdb.Kvdb
+}
+
+const (
+	opType    = "operation"
+	errString = "error"
+	output    = "output"
+)
+
+// New constructs a new kvdb.Kvdb.
+func NewLogWrapper(
```

Comment:
This should be configurable so that these logs can be routed to a file, and we can use it as a kvdb audit log.

Reply:
ok.